x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -8196,7 +8196,6 @@
     "xpack.ingestManager.agentConfigForm.systemMonitoringText": "システムメトリックを収集",
     "xpack.ingestManager.agentConfigForm.systemMonitoringTooltipText": "このオプションを有効にすると、システムメトリックと情報を収集するデータソースで構成をブートストラップできます。",
     "xpack.ingestManager.agentConfigList.actionsColumnTitle": "アクション",
-    "xpack.ingestManager.agentConfigList.actionsMenuText": "開く",
```

Comment:
these two translation files may need to be reverted if you revert the change to the i18n key